mcs/class/referencesource/mscorlib/system/runtime/interopservices/runtimeenvironment.cs

```diff
@@ -151,6 +151,18 @@ static String GetRuntimeDirectoryImpl()
             }
         }
 
+#if MONO
+        public static IntPtr GetRuntimeInterfaceAsIntPtr(Guid clsid, Guid riid)
+        {
+            throw new NotSupportedException();
+        }
+
+        public static object GetRuntimeInterfaceAsObject(Guid clsid, Guid riid)
+        {
+            throw new NotSupportedException();
+        }
+#endif
```

Comment:
That doesn't work when FEATURE_COMINTEROP is not defined (like on monotouch) and adding || MONO somehow felt ugly. I'll change it though :)